Generator.Core/Generator.Core.fsproj

```diff
@@ -38,8 +39,11 @@
 		<Compile Include="CodeOut\ArrayWriter.fs" />
 	</ItemGroup>
 
-
 	<ItemGroup>
+		<!--
+		TODO: Analyzer and source generator assemblies can reference either Microsoft.CodeAnalysis.CSharp or
+		Microsoft.CodeAnalysis.VisualBasic, but must not reference both.
+		-->
 		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.1.0-1.final" />
 		<PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="4.1.0-1.final" />
```

Comment:
📝 You'll want to create three projects here:

* Generator.Core: References Microsoft.CodeAnalysis.Common and is included in both C# and VB scenarios
* Generator.Core.CSharp: Extends Generator.Core and only applies in C# scenarios
* Generator.Core.VisualBasic: Extends Generator.Core and only applies in VB scenarios

I already configured the package project to split the three assemblies into the correct folders.